contrib/remote/containers.conf

```diff
@@ -0,0 +1,13 @@
+# The containers configuration file specifies all of the available configuration
+# command-line options/flags for container engine tools like Podman
+# but in a TOML format that can be easily modified and versioned.
+
+[engine]
+
+# Default Remote URI to access the Podman service.
+# Examples:
+#  rootless "unix://run/user/$UID/podman/podman.sock" (Default)
+#  rootfull "unix://run/podman/podman.sock.(Default)
```

Comment:
do these hold for Windows?  IDK, just asking.

Reply:
No the Unix examples should be removed.